card/mod/core/spec/set/all/fetch_helper_spec.rb

```diff
@@ -0,0 +1,39 @@
+# -*- encoding : utf-8 -*-
+
+RSpec.describe Card::Set::All::FetchHelper do
+  let :retrieve do
+    Card.send(:retrieve_existing, mark, {}).map(&:class)
+  end
+
+  let :retrieve_from_trash do
+    Card.send(:retrieve_existing, mark, look_in_trash: true).map(&:class)
```

Comment:
Is the `map(&:class)` needed here?
Since you don't use the return value `let` isn't of much use here. Methods would make more sense to me.